tests/integration/test_api.py

```diff
@@ -357,6 +358,106 @@ def test_get_tasks(self):
         self.assertEqual(1, len(response_get.json()["results"]))
         self.assertEqual(initial_item_id, response_get.json()["results"][0]["id"])
 
+    def test_get_tasks_filters(self):
```

Comment:
Ideally, these should be split up into different tests (each API call is a different test)